mne/tests/test_report.py

```diff
@@ -41,6 +43,10 @@
                                'data'))
 evoked_fname = op.join(base_dir, 'test-ave.fif')
 
+# Two example figures
+fig1 = plt.plot([1, 2], [1, 2])[0].figure
+fig2 = plt.plot([3, 4], [3, 4])[0].figure
+
```

Comment:
Drawing plots takes time. Better to generate two that can be used by all test functions.